clang/include/clang/Driver/Driver.h

```diff
@@ -658,6 +661,19 @@ class Driver {
   static void getDefaultModuleCachePath(SmallVectorImpl<char> &Result);
 
   bool getOffloadStaticLibSeen() const { return OffloadStaticLibSeen; };
+
+  /// addFPGATempDepFile - Add a file to be added to the bundling step of
+  /// an FPGA object.
+  void addFPGATempDepFile(const std::string &Name,
+                          const std::string &JA) const {
```

Comment:
Personally, I'm not sure if `JA` is explanatory (applicable to the getter method as well)

Reply:
It isn't.  cut-n-paste naming (was associated with JobAction, but this isn't true here).  I will update to a better naming.

Reply:
Updated Names.